src/main/resources/hudson/plugins/jira/Messages.properties

```diff
@@ -33,3 +33,4 @@ CommentStep.Descriptor.DisplayName=JIRA: Add a comment to issue(s)
 SearchIssuesStep.Descriptor.DisplayName=JIRA: Search issues
 IssueSelectorStep.Descriptor.DisplayName=JIRA: Issue selector
 JiraCreateIssueNotifier.DisplayName=JIRA: Create issue
+FindIssuesAtCommitLogsStep.Descriptor.DisplayName=JIRA: find all issues referenced by the commit logs of this build
```

Comment:
I know what you meant now. In previous Workflow versions only the description was shown in the dropdown, true. But now (due to [this](https://github.com/jenkinsci/workflow-plugin/commit/577f2acb256c0c91eaa81abe3177b4248bbaa059)) the elements in that dropdown list are ordered by the step function name. This was introduced in `workflow:1.11`, so as long as you depending on 1.12 the `JIRA:` fragment could be removed safely.